<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocalVariableDialog</class>
 <widget class="QDialog" name="LocalVariableDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>372</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>677</width>
    <height>372</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>677</width>
    <height>372</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="m_pButtons">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>340</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>331</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>Integers:</string>
   </property>
   <widget class="CustomVariableIntPageWidget" name="m_pInts" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>311</width>
      <height>132</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>Bools:</string>
   </property>
   <widget class="CustomVariableBoolPageWidget" name="m_pBools" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>311</width>
      <height>132</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>331</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>Floats:</string>
   </property>
   <widget class="CustomVariableFloatPageWidget" name="m_pFloats" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>311</width>
      <height>132</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>331</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>Strings:</string>
   </property>
   <widget class="CustomVariableStringPageWidget" name="m_pStrings" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>311</width>
      <height>132</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomVariableBoolPageWidget</class>
   <extends>QWidget</extends>
   <header>customvariableboolpagewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CustomVariableIntPageWidget</class>
   <extends>QWidget</extends>
   <header>customvariableintpagewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CustomVariableStringPageWidget</class>
   <extends>QWidget</extends>
   <header>customvariablestringpagewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CustomVariableFloatPageWidget</class>
   <extends>QWidget</extends>
   <header>customvariablefloatpagewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_pButtons</sender>
   <signal>accepted()</signal>
   <receiver>LocalVariableDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pButtons</sender>
   <signal>rejected()</signal>
   <receiver>LocalVariableDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
